deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml

```diff
@@ -70,11 +76,6 @@ objects:
     - "*"
   - apiGroups:
     - ""
-    resources:
-    - secrets
-    verbs:
-    - create
-    - delete
```

Comment:
let's agree with the rest of the AppStudio team that no one needs that anymore before we drop this permission.
cc @sbose78

Reply:
I've re-added the secrets for the time being. Let's deal with their potential removal in another PR. Thanks for pointing it out!

Reply:
I haven't verified this isn't needed anymore.